src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -0,0 +1,67 @@
+import math
+
+def get_lot_size(smallest_decimal_unit: float, coin_decimals: int) -> int:
+    """
+    Given a decimal representation of the smallest possible unit and the
+    whole unit's decimals (e.g. ETH has 18 decimals), return the lot size.
+
+    Parameters:
+    * `smallest_decimal_unit`: Decimal (e.g. 0.001 for one thousandth) of
+      the smallest possible unit relative to a whole unit.
+    * `coin_decimals`: The number of decimals one whole unit of the coin
+      has (e.g. USDC has 6 decimals, ETH has 18).
+    """
+    return math.ceil(10**(coin_decimals+math.log10(smallest_decimal_unit)))
+
+def get_tick_size(smallest_decimal_unit: float, coin_decimals: int) -> int:
+    """
```

Comment:
Here again I suggest `get_tick_size_integer`, using string input then cast to decimal.Decimal

Also do type checking to enforce that what should be an int is an int

Reply:
I think the type checking python does (which I think is none) is fine given there's type hints. We shouldn't be type checking in type-hinted functions.